package.json

```diff
@@ -24,13 +24,12 @@
     "lint:base": "eslint --rule \"prettier/prettier: 2\"",
     "lint": "npm run lint:base -- .",
     "lint:fix": "npm run lint -- --fix",
-    "pretest": "npm run lint",
-    "test": "jest --coverage",
+    "test": "jest",
```

Comment:
Why this change?

Reply:
Was having some troubles with it initially on my Windows box but works fine now. Will Revert.